app/eq.py

```diff
@@ -128,6 +130,29 @@ async def build(self):
 
         return self._payload
 
+    @staticmethod
+    def encrypt_response_id(questionnaire_id: str):
+        """
+        Encrypts a response_id using the following format: <response_id><first 16 characters of sha256 signature>.
+        Utilises EQ_SALT environment property as salt.
+        Implements the HMAC algorithm as described by RFC 2104.
+
+        :param questionnaire_id: the questionnaire_id utilised to encrypt the response_id
+        :return: the encrypted response_id
+        """
+        try:
+            eq_salt = app.config.BaseConfig.EQ_SALT
```

Comment:
Don't call direct from BaseConfig, it risks not using the correct value in all environments - use app['EQ_SALT'] instead (examples for other config values at top of this file)